wishlist/wishlist.css

```diff
@@ -0,0 +1,186 @@
+
+.page-container{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+}
+.navbar{
+    margin-bottom: 5px;
+    padding: 1.3rem var(--spacing3) ;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-bottom: 6px;
+    border-bottom: 1px solid rgb(207, 207, 207);
+}
+.brand-name{
+    font-size: var(--fs3);
+}
+/* searchbar */
+.search-bar{
+    margin-left: 10px;
+    margin-right: 10px;
+    display: flex;
+    align-items: center;
+    background-color: white;
+    border: 1px solid var(--primary-color);
+}
+
+.search-bar i{
+    margin-left: 4px;
+    color: gray;
+}
+.search-bar input{
+    padding: 10px 10px;
+    border-radius: 2px;
+    width: 100%;
+    border: none;
+    
+}
+.search-bar-desktop{
+    display: none;
+}
+/* hamburger menu */
+.hamburger-menu{
+    padding: 0 0.5rem 0 0;
+    font-size: 1.2rem;
+    display: flex;
+    align-items: center;
+}
+.navbar a {
+    text-decoration: none;
+    color: var(--primary-color);
+}
+.navbar a h1{
+    display: inline;
+    margin-left: var(--spacing3);
+    padding:1rem 0.5rem 1rem 0;
+}
+
+.action-icons{
+    display: flex;
+    margin-left: auto;
+    align-items: center;
+    gap: var(--spacing6);
+    margin-right: 10px;
+}
+.action-icons .icon-container{
+    font-size: 1rem;
+}
+.action-icons .icons{
+    padding: 0.5rem;
+}
+
+/* main content */
+.main-content{
+    /* border: 1px solid black; */
+    display: flex;
+    justify-content: center;
+    font-size: larger;
+    /* border: 1px solid black; */
+}
+/* wishlist */
+.wishlist-section{
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+.wishlist-section .product-card{
+    width: 10rem;
+    margin-top: 1rem;
+    margin-bottom: 0.8rem;
+    margin-right: 9.75px;
+    margin-left: 9.75px;
+    padding-bottom: 0.5rem;
+}
+
+.wishlist-section .product-card .btn{
+    font-size: 1rem;
+}
+
```

Comment:
I personally like to remove all empty lines in-between the code blocks. I keep only one line in between for readability .
If there is a reason, I keep more empty lines.